en/dev_setup/dev_env_linux_ubuntu.md

```diff
@@ -51,9 +51,15 @@ ROS users must follow the instructions for: [ROS/Gazebo](#rosgazebo). <!-- ROS i
 To install the toolchain:
 
 1. [Download PX4 Source Code](../dev_setup/building_px4.md):
+
    ```bash
    git clone https://github.com/PX4/PX4-Autopilot.git --recursive
    ```
+
+   :::note
```

Comment:
I added these notes in our setup. Generally the head revision setup scripts changes slowly, but they do change. Usually the changes are backward compatible. This note just exists so that it is clear that you might need to use the version of the script associated with the release.

Reply:
Make perfectly sense!